server/src/main/java/org/apache/iotdb/db/utils/datastructure/TVList.java

```diff
@@ -46,7 +47,7 @@
   /**
    * this field is effective only in the Tvlist in a RealOnlyMemChunk.
    */
-  private long timeOffset = Long.MIN_VALUE;
+  private List<Pair<Long, Long>> deletionList = new ArrayList<>();
```

Comment:
Deletion operation is rare.
Put `new ArrayList()` to `delete()`, add check whether the list == null before using it.